cmd/launcher/launcher.go

```diff
@@ -558,8 +575,8 @@ func runOsqueryVersionCheck(ctx context.Context, logger log.Logger, osquerydPath
 		return
 	}
 
-	level.Debug(logger).Log("msg", "checked osqueryd version",
-		"version", outTrimmed,
+	slogger.Log(ctx, slog.LevelDebug,
+		"checked osqueryd version",
 		"execution_time_ms", executionTimeMs,
 		"osqueryd_path", osquerydPath,
```

Comment:
We don't want to include `"version", outTrimmed,`?

Reply:
good catch, updated